lib/cartopy/crs.py

```diff
@@ -915,7 +919,7 @@ def _project_linear_ring(self, linear_ring, src_crs):
         if rings:
             multi_line_string = sgeom.MultiLineString(line_strings)
 
-        return rings, multi_line_string
+        return sgeom.GeometryCollection([*rings, multi_line_string])
```

Comment:
Update the docstring of this function to indicate that it is returning a GeometryCollection now.